index.html

```diff
@@ -66,7 +69,7 @@ <h2 class="banner-heading text-center font-heading cap">we work with our clients
         <section class="why-us">
             <div class="why-us_content">
                 <h2 class="sub-heading font-heading text-center sub-heading headings-common_styles">Why Work With Us</h2>
-                <p class="paragraph-sec paragraph-common width77 center-by_margin__auto margin-top-medium text-center">We specialize in softwares,web development and web design. We provide tech support for all sorts of progams from xd,figma,html,css,javascript, react and more.</p>
+                <p class="paragraph-sec paragraph-common width77 center-by_margin__auto margin-top-medium text-center">We specialize in softwares, Web Development and Web Design. We provide tech support for all sorts of progams from XD, Figma, HTML, CSS, JavaScript, React and more.</p>
```

Comment:
**Update to:**
We specialize in Software, Web and Mobile Application Development and Design.
instead of: _We provide tech support for all sorts of progams from XD, Figma, HTML, CSS, JavaScript, React and more._

Write:
We use the latest and secure tools such as React.js, Node/JavaScript, Figma, Java, Spring Boot, Relational and Non-relational databases as well as various Cloud Platforms and more, ensuring that our clients receive Robust, Secure and Efficient solutions for their problems.